"Cannot read property 'indexOf' of undefined" when checkout on ungit 0.10.3

Hi, I would like to create an issue, "Something went wrong, reload the page to start over." comes up when I make a ”checkout" on ungit 0.10.3. Following is the stack I got from Chrome:

server.js:220 Cannot read property 'indexOf' of undefinedServer._onUnhandledBadBackendResponse @ server.js:220
server.js:225 Error: Backend error: Cannot read property 'indexOf' of undefined
    at Server.query (http://0f7ca0ff-5ff4-458e-8ceb-2c0813528a61-8448.microtools.swarm/js/ungit.js:6325:25)
    at Server.post (http://0f7ca0ff-5ff4-458e-8ceb-2c0813528a61-8448.microtools.swarm/js/ungit.js:6311:8)
    at GraphActions.Checkout.perform (http://0f7ca0ff-5ff4-458e-8ceb-2c0813528a61-8448.microtools.swarm/plugins/graph/graph.bundle.js:291:15)
    at GraphActions.Checkout.GraphActions.ActionBase.doPerform (http://0f7ca0ff-5ff4-458e-8ceb-2c0813528a61-8448.microtools.swarm/plugins/graph/graph.bundle.js:93:8)
    at HTMLDivElement.<anonymous> (http://0f7ca0ff-5ff4-458e-8ceb-2c0813528a61-8448.microtools.swarm/js/ungit.js:28313:62)
    at HTMLDivElement.dispatch (http://0f7ca0ff-5ff4-458e-8ceb-2c0813528a61-8448.microtools.swarm/js/ungit.js:7213:9505)
    at HTMLDivElement.y.handle (http://0f7ca0ff-5ff4-458e-8ceb-2c0813528a61-8448.microtools.swarm/js/ungit.js:7213:6288)
Backend stacktrace:
TypeError: Cannot read property 'indexOf' of undefined
    at /usr/lib/node_modules/ungit/src/git-promise.js:242:15
    at tryCatcher (/usr/lib/node_modules/ungit/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/usr/lib/node_modules/ungit/node_modules/bluebird/js/release/promise.js:502:31)
    at Promise._settlePromise (/usr/lib/node_modules/ungit/node_modules/bluebird/js/release/promise.js:559:18)
    at Promise._settlePromise0 (/usr/lib/node_modules/ungit/node_modules/bluebird/js/release/promise.js:604:10)
    at Promise._settlePromises (/usr/lib/node_modules/ungit/node_modules/bluebird/js/release/promise.js:683:18)
    at Async._drainQueue (/usr/lib/node_modules/ungit/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/usr/lib/node_modules/ungit/node_modules/bluebird/js/release/async.js:148:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/usr/lib/node_modules/ungit/node_modules/bluebird/js/release/async.js:17:14)
    at tryOnImmediate (timers.js:543:15)
    at processImmediate [as _immediateCallback] (timers.js:523:5)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](http://semver.org/) and
 ### Fixed
 - File diff firing increasing number of events longer it survives.
 - Fix missing ungit logo. [#812](https://github.com/FredrikNoren/ungit/issues/812)
+- Fix when stash output is empty [#818](https://github.com/FredrikNoren/ungit/issues/818)
 
 ## [0.10.3](https://github.com/FredrikNoren/ungit/compare/v0.10.2...v0.10.3)
 

--- a/source/git-promise.js
+++ b/source/git-promise.js
@@ -259,7 +259,7 @@ git.stashExecuteAndPop = (commands, repoPath, allowError, outPipe, inPipe, timeo
         throw err;
       }
     }).then((result) => {
-      if (result.indexOf('No local changes to save') != -1) {
+      if (!result || result.indexOf('No local changes to save') != -1) {
         hadLocalChanges = false;
       }
       return git(commands, repoPath, allowError, outPipe, inPipe, timeout);

